--- foo	2005-09-24 14:19:27.937500000 +1000
+++ src/test/java/org/openejb/corba/compiler/PortableStubCompilerTest.java	2005-09-24 14:19:47.015625000 +1000
@@ -64,15 +64,15 @@
     private static final File basedir = new File(System.getProperty("basedir", System.getProperty("user.dir")));
 
     public void testBeanPropertiesNameMangler() throws Exception {
-        assertMangling("src/test-resources/beanPropertiesNameMangler.properties", BeanProperties.class);
+        assertMangling("src/test/resources/beanPropertiesNameMangler.properties", BeanProperties.class);
     }
 
     public void testBasicNameMangler() throws Exception {
-        assertMangling("src/test-resources/nameMangler.properties", Foo.class);
+        assertMangling("src/test/resources/nameMangler.properties", Foo.class);
     }
 
     public void testSpecialNameMangler() throws Exception {
-        assertMangling("src/test-resources/specialNameMangler.properties", Special.class);
+        assertMangling("src/test/resources/specialNameMangler.properties", Special.class);
     }
 
     private void assertMangling(String propertiesFile, Class intf) throws IOException {
